packages/dendron-plugin-views/src/components/Seeds.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  CheckCircleOutlined,
+  DisconnectOutlined,
+  DownloadOutlined,
+  GlobalOutlined,
+} from "@ant-design/icons";
+import {
+  DMessageSource,
+  SeedBrowserMessage,
+  SeedBrowserMessageType,
+} from "@dendronhq/common-all";
+import { Tooltip } from "antd";
+import * as React from "react";
+import { postVSCodeMessage } from "../utils/vscode";
+
+/**
+ * Component for the GoToSite button on a seed browser card
+ * @param url - URL to open
+ * @param inVscode - set if within vs code webview, do not set for browser.
+ * @returns
+ */
+export function GoToSiteButton({
+  url,
+  inVscode,
+}: {
+  url: string | undefined;
+  inVscode: boolean;
+}) {
+  function onClick() {
+    if (url) {
+      // If we're in VSCode, the webview does not allow popups, so send a
+      // message back to the plugin and open the link from within the plugin
+      if (inVscode) {
+        postVSCodeMessage({
+          type: SeedBrowserMessageType.onOpenUrl,
+          data: { data: url },
+          source: DMessageSource.webClient,
+        } as SeedBrowserMessage);
+      } else {
+        window.open(url);
+      }
+    }
+  }
+
+  if (url) {
+    return (
+      <Tooltip placement="top" title="Go to Site">
+        <GlobalOutlined key="website" onClick={onClick} />
```

Comment:
Not the point of the PR, but if it's not too much work could we also switch these to put the text next to the icon, instead of having it be a tooltip? So:

```jsx
<Text><GlobalOutlined ... /> Go to Site </Text>
```

Same for the other buttons. That's more accessible for screen readers, and more intuitive for everyone I think. But again not the point of this PR so feel free to ignore this.

Reply:
This is pretty easy to do but I think the reason behind keeping the tooltip is to keep the design clean since some of the messages are long `Site Unavailable `Add to workspace` and `Already in Workspace`